server/plugin/configuration_test.go

```diff
@@ -21,27 +21,22 @@ func TestIsValid(t *testing.T) {
 				ServiceNowOAuthClientSecret: "mockServiceNowOAuthClientSecret",
 				EncryptionSecret:            "mockEncryptionSecret",
 				WebhookSecret:               "mockWebhookSecret",
+				ChannelCacheSize:            1000,
```

Comment:
not a compulsory change but if we giving this mock values its better to give the actual default values of respective settings